tests/std/tests/GH_002030_asan_annotate_string/test.cpp

```diff
@@ -1917,17 +1919,32 @@ void test_gh_3955() {
     assert(s == t);
 }
 
-int main() {
-    run_allocator_matrix<char>();
+void test_gh_5251() {
+    // GH-5251 <string>: ASan annotations do not prevent writing to allocated
+    // but uninitialized basic_string memory
+    std::basic_string<char> myString;

```

Comment:
Nitpick: This test is already `using namespace std;`, so we shouldn't use redundant `std::` qualification.

Suggestion: This can just be `string`.